Automattic/wp-calypso
# Customer Home: Release to 100% of new users

#### Changes proposed in this Pull Request

This is the first stage of removing the customer home A/B test logic.
The A/B test is checked, rather than assigned, and if it doesn't exist
then the user is assumed to be in the `show` variant.

Only users not eligible for the checklist menu item or in the `hide`
variant will not see the customer home page.

#### Testing instructions

Run this branch, either locally or via calypso.live

Make sure that you're not participating in the A/B test by running:

`localStorage.setItem( 'ABTests', JSON.stringify( Object.fromEntries( Object.entries( JSON.parse( localStorage.getItem( 'ABTests' ) ) ).filter( function( e ) { return ! e[0].startsWith('customerHomePage') } ) ) ) )`

in your browser console.

Also remove the associated user attributes by running:

`$wpdb->query( "DELETE FROM wp_user_attributes WHERE user_id =` __USER_ID__ `AND meta_key like 'abtest_customerHomePage%'" );`

Substituting your user ID.

Now check that any site that did have the customer home page can still access it, and that any new site you create has customer home enabled.

Additionally, put yourself in the `hide` test group (using the ABTEST menu under the environment badge) and check that the customer home is hidden on sites created before 6th August or replaced with the Checklist for sites created after that date.
